### PR TITLE
Refactor Codebase

### PR DESCRIPTION
**Pull request by andela-anwocha**
#### What does this PR do?
Modify code base according to reviews.

#### Description of Task to be completed?
Refactor code to conform to Ruby style guide.

#### How should this be manually tested?
Tests can be carried out using: ```bundle exec rspec spec```

### CODE REVIEW

File: lib/concerns/class_methods.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    def all
+      @all
+    end
+
+    def destroy_all
+      @all.clear
+    end
+  end
+end
```
Comment (andela-anwocha) [inline lib/concerns/class_methods.rb]: Separated modules into different files

File: lib/concerns/instance_methods.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    def add_song(song)
+      unless @songs.include?(song)
+        @songs << song
+        song.send(self.class.name.downcase + '=', self)
+      end
+    end
+  end
+
+end
```
Comment (andela-anwocha) [inline lib/concerns/instance_methods.rb]: Separated modules into different files

File: lib/message.rb [hunk 3]
```diff
@@ -31,7 +31,7 @@ def self.error_message
     EOT
   end
 
-  def self.info(msg, _color = :yellow)
-    puts msg.colorize(_color)
+  def self.info(msg, color = :yellow)
+    puts msg.colorize(color)
   end
 end
```
Comment (andela-anwocha) [inline lib/message.rb]: Changed `_color` to `color`

File: spec/012_musiclibrary_cli_spec.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@
     expect(MusicLibraryController).to receive(:new).and_return(music_libray_controller)
     expect(music_libray_controller).to receive(:gets).and_return("list songs", "exit")
 
-    output = capture_puts {run_file("./bin/musiclibrary")}
+    output = capture_puts {run_file("./bin/music_library")}
```
Comment (andela-anwocha) [inline spec/012_musiclibrary_cli_spec.rb]: Renamed `musiclibrary` file to `music_library`

File: lib/concerns/common.rb [hunk 5]
```diff
@@ -1,34 +0,0 @@
...
-    end
-
-    def add_song(song)
-      unless @songs.include?(song)
-        @songs << song
-        song.send(self.class.name.downcase + '=', self)
-      end
-    end
-    # Contains common Class methods for all models
-    module ClassMethods
```
Comment (andela-anwocha) [inline lib/concerns/common.rb]: Separated module into a different file of its own

File: lib/concerns/instance_methods.rb (hunk 2 above)
Comment (andela-bmakinwa) [inline lib/concerns/instance_methods.rb]: 👍

**Pull request merged**